static/src/javascripts/projects/common/modules/ui/selection-sharing.js

```diff
@@ -0,0 +1,173 @@
+// @flow
+import bean from 'bean';
+import Rangefix from 'rangefix';
+import $ from 'lib/$';
+import config from 'lib/config';
+import { hasTouchScreen } from 'lib/detect';
+import mediator from 'lib/mediator';
+import template from 'lodash/utilities/template';
+import { inlineSvg } from 'common/views/svgs';
+import debounce from 'lodash/functions/debounce';
+
+const sharingTemplate = `
+    <div class="selection-sharing">
+        <ul class="social u-unstyled u-cf" data-component="social-selection">
+            <li class="social__item" data-link-name="twitter">
+                <a  class="rounded-icon social-icon social-icon--twitter js-selection-twitter"
+                    data-link-name="social-selection"
+                    target="_blank"
+                    href="#">
+                    <span class="u-h">Share on Twitter</span>
+                    <%= twitterIcon %>
+                </a>
+            </li>
+            <li class="social__item" data-link-name="email">
+                <a  class="rounded-icon social-icon social-icon--email js-selection-email"
+                    data-link-name="social-selection"
+                    target="_blank"
+                    href="#">
+                    <span class="u-h">Share via Email</span>
+                    <%= emailIcon %>
+                </a>
+            </li>
+        </ul>
+    </div>
+`;
+
+const twitterIcon = inlineSvg('shareTwitter', ['icon', 'centered-icon']);
+const emailIcon = inlineSvg('shareEmail', ['icon', 'centered-icon']);
+const $body = document.body;
```

Comment:
The `$` at the start of this constant name is misleading as it implies the object is a `bonzo` instance. Can we instead inline `document.body` everywhere it is used?